internal/config/app_config_sort.go

```diff
@@ -112,6 +115,7 @@ func (c *genericConfig) sortVars(ctx *hcl.EvalContext) ([]*analyzedPair, error)
 				Name:     key,
 				Refs:     refs,
 				Internal: m.internal,
+				Path:     m.path,
```

Comment:
Is this preferred for it's explicitness?

    Path: m.prefix == "config.file."

Would also work here right?

Reply:
It would, but it's better to have been explicit about the choice.